src/pages/api/__tests__/github-load-notes.test.ts

```diff
@@ -0,0 +1,15 @@
+import {parseMdxNotesFile} from '../github-load-notes'
+
+// this had to use an async/await because the mdx compiler
+// is set up to be async and return a promise
+test('properly parses a markdown text string', async () => {
+  const markdownNotes = `<TimeStamp start="0:44" end="0:50">
+
+This is a **note**.
+
+</TimeStamp>
+`
+  const parsedNotes = parseMdxNotesFile(markdownNotes)
+
+  await expect(parsedNotes).resolves.toEqual('not the answer')
```

Comment:
because our `parseMdXNotesFile` method returns a `Promise` we can use await so that our test can understand to wait for the promise to resolve before checking the value